src/style.css

```
@@ -0,0 +1,127 @@
+body {
+  background-color: #fff;
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  margin-left: 0.5%;
+  margin-right: 0.5%;
+  margin-top: 0.25%;
+  margin-bottom: 0.25%;
+}
+
+.myimg {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 33px;
```

center image horizontally with flex

```suggestion
  margin-bottom: 33px;
```